src/utils/index.ts

```diff
@@ -0,0 +1,62 @@
+import { Currency, Token } from '@uniswap/sdk-core'
```

Comment:
Thoughts on the naming of this file? I had trouble coming up with a name. These functions are only imported and used in `swapRouter.ts` so `index.ts` made some sense. Also, would we want to export this from the package? Right now I have it as not exported.